src/common/components/menuDropdown/__tests__/MenuDropdown.test.tsx

```diff
@@ -105,7 +105,7 @@ describe('MenuDropdown component', () => {
 
     clickOnToggleButton();
 
-    items.forEach(async (item, index) => {
+    items.forEach((item, index) => {
       getItemAtIndex(index).click();
```

Comment:
Isn't the click async here?

Reply:
Revamped the MenuDropdown tests in 969e75148149d1da502c1941af8a597f2dfa197c, now it should be ok.